base/docs/basedocs.jl

```diff
@@ -2909,9 +2909,9 @@ Base.swapproperty!
 """
     modifyproperty!(x, f::Symbol, op, v, order::Symbol=:not_atomic)
 
-The syntax `@atomic max(a().b, c)` returns `modifyproperty!(a(), :b,
-max, c, :sequentially_consistent))`, where the first argument must be a
-`getfield` expression and is modified atomically.
+The syntax `@atomic op(x.f, v)` (and its equivalent `@atomic x.f op v`) returns
+`modifyproperty!(x, :f, op, v, :sequentially_consistent)`, where the first argument
+must be a `getproperty` expression and is modified atomically.
```

Comment:
`swapproperty!` also calls it a `getfield` expression

Reply:
Do you want to keep calling it `getfield` expression?

Reply:
Either way, just as long as we are consistent

Reply:
Okay, let's go with `getproperty` then. I think `getproperty` is more familiar with general users.